### PR TITLE
Integrate ras rm print file

### PR DESCRIPTION
**Pull request by warrenbailey**
# Motivation and Context

Integration with the RAS/RM printfile

# What has changed

Removes the following functionality:
1. SFTP 
1. GCS
1. Freemaker templating
1. Action Feedback processing 

Also simplifies the spring integration patterns

Includes a database change to drop the template mapping table as it will no longer be needed.

# How to test?

# Links

### CODE REVIEW

**Review by jterryons — APPROVED** [review]

File: src/main/java/uk/gov/ons/ctp/response/action/export/service/FilenamePrefix.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import java.util.Map;
+
+/** This class replaces the template mappings database */
+public class FilenamePrefix {
+  private static final Map<String, String> mappings = new HashMap<>();
+
+  static {
+    mappings.put("BSNOT", "BSNOT");
+    mappings.put("BSREM", "BSREM");
+    mappings.put("BSNL", "BSNOT");
```
Comment (insacuri) [inline src/main/java/uk/gov/ons/ctp/response/action/export/service/FilenamePrefix.java:13]: More of a question then a comment, what are BSNL and BSRL?

Comment (warrenbailey) [reply src/main/java/uk/gov/ons/ctp/response/action/export/service/FilenamePrefix.java:13]: Business Survey Notification Letter
Business Survey Notification Email

**Pull request merged**